pages/pil/modules_exempt/content/index.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Not sure of the naming convention for directories with multiple words - for files they should be `some-thing.js` rather than `some_thing.js` but not sure if this applies to directories as well? 

Ping @lennym @joefitter

Reply:
see my comment above :) yeah hyphens

Reply:
lint shoulda caught that but I'm guessing it maybe didn't as its a directory, not a filename